bcgov/foi-flow
# Migration from Silver to Gold realm

Test Cases Executed:

- Login to application
- Logout successfully (Session Invalidation)
- Create Request
- State transition
- Add comment
- Upload Documents
- Watch/UnWatch Request
- Verify payment.
- Axis Sync
- Submission from angular app
- Add/Remove Divisions
- Add Extension to Request.
